index.js

```diff
@@ -21,7 +21,10 @@
 //. import {env, FutureType} from 'fluture-sanctuary-types/index.js';
 //. import {resolve} from 'fluture/index.js';
 //.
-//. const S = sanctuary.create ({checkTypes: true, env: S.env.concat (env)});
+//. const S = sanctuary.create ({
+//.   checkTypes: true,
+//.   env: sanctuary.env.concat (env),
+//. });
```

Comment:
I think that for many new users, this may be the first time they are introduced to `sanctuary.create`. For that reason, it might be nice to add a comment by `checkTypes: true` that they need to configure it such that type checking is disabled in production. What do you think @davidchambers ?

Reply:
Or maybe I should just put `process.env.NODE_ENV !== 'production'` there instead of `true`.

Reply:
> Or maybe I should just put `process.env.NODE_ENV !== 'production'` there instead of `true`.

Good idea.